### PR TITLE
Feature/normalize phonenumbers

### PR DESCRIPTION
**Pull request by lukasender**
## notes for the reviewer:

the following tables (and fields) need to be migrated:

- ``users.mobile``
- ``supporters.relations.user.mobile``
- ``petitions.relations.owner.mobile``
- ``confirmations.data['mobile'] where handler == 'petition_sms'``
- ``confirmations.data['user']['mobile'] where handler == 'support_sms``

# we don't need the "complex" (Supporter.id & Confirmation.data['supporter']) migration

~~if we need to keep **existing, unused** ``confirmations``, we also need to migrate:~~

- ~~``confirmations.data['supporter']`` where ``confimartions.handler == 'supporter_confirm_email'``.~~
- ~~``supporters.id``~~

~~in that case, ``confirmations.data['supporter']`` is a ``Supporter.id`` (which, for session users, contains a phone number).
we then need to migrate ``Supporter.id`` as well. however, ``Supporter.id`` is a primary key (and thus used for routing) and cannot be altered. the respective document needs to be fetched, process, deleted and inserted again with the migrated ``Supporter.id``~~

~~on production, the second part of the migration would potentially affect ``1868`` confirmation objects (potentially because "data" is "not indexed" and thus we can't filter more detailed)~~

### CODE REVIEW

File: iris/service/content/user/document.py [hunk 1]
```diff
@@ -57,6 +81,10 @@ class User(Document):
         default=''
     )
 
+    @mobile.setter
+    def set_moblie(self, value):
```
Comment (jukart) [inline iris/service/content/user/document.py]: typo: set_mobile

File: iris/service/content/petition/sm.py [hunk 2]
```diff
@@ -218,8 +218,13 @@ def support_petition(self, **kwargs):
                 except ValueError:
                     # a ValueError means that confirmation failed
                     pass
-                if msg and data['user']['mobile'] == mobile:
-                    # We trust the mobile because the virification token is
+                # FIXME: data['user']['mobile'] == 'mobile' is always True
```
Comment (jukart) [inline iris/service/content/petition/sm.py]: Strange code which is probably a remnant.
`if msg:` seems to do the work

File: CHANGES.rst [hunk 3]
```diff
@@ -5,6 +5,10 @@ Changes For IRIS Service
 unreleased
 ==========
 
+ - Normalise 'mobile'
+   DEPLOY:
+        Perform 'mobile phone normalise migration'
```
Comment (jukart) [inline CHANGES.rst:10]: What needs to be done to do this?

Comment (lukasender) [reply CHANGES.rst:10]: added a new line below

**Comment by jukart** [comment]
ack

**Pull request merged**